labs/a3.md

```diff
@@ -78,14 +78,24 @@ sudo apt install --no-install-recommends qemu-kvm qemu-utils libvirt-bin virtins
 
 ### acquire installation media
 
-Next, we need to get the installation media. Download this, as well as
-a signature that we will use to verify the authenticity of the media,
-using the following commands:
+Next, we need to get the installation media. Go to
+https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/ to find the latest Arch
+release. Using the `wget` command within your VM, download the `.iso` file found
+within this directory as well as the `.iso.sig` signature file that we will use
+to verify the authenticity of the `.iso` file. For example, the current release
+at the time of writing is `2020.06.01`, so you would run the following commands:
 
 ```sh
-wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/2019.09.01/archlinux-2019.09.01-x86_64.iso'
-wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/2019.09.01/archlinux-2019.09.01-x86_64.iso.sig'
+wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/archlinux-2020.06.01-x86_64.iso'
```

Comment:
This is outdated already, so should we do `archlinux-$(date +"%Y.%m").01-x86_64.iso` instead? This is a3 so we might be able to assume they understand that command.

Reply:
Good point about understanding the command, but I'm not sure how timely Arch releases actually are. There's probably a narrow window where that command won't work given it uses your local time which may be in a different timezone. Maybe:

```
wget -r -l1 --no-parent -A 'archlinux-*-x86_64.iso' 'https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/'
wget -r -l1 --no-parent -A 'archlinux-*-x86_64.iso.sig' 'https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/'
```

I had to Google to find all those flags though which was more work than looking for the actual ISO and signature files in https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/. I also think there's some value in slowing people down a bit to show them "this is what a software mirror is in case you didn't know" because if I were in a hurry and a working command was given I would just copy-paste and move on.